Add an index page to the docu and change readme of the repo. (#35)
echemdb/echemdb: doc/usage/database_interactions.md

+++

> A database can be generated from a remote or local source. 

Reviewer: ```suggestion
A database can be generated from a remote or a local source. 
```

> In the example the database is created from the available entries shown on [echemdb.org](https://www.echemdb.org).

Reviewer: ```suggestion
You can create a database from the entries on [echemdb.org](https://www.echemdb.org):
```
